src/Sulu/Bundle/EventLogBundle/phpunit.xml.dist

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit bootstrap="../TestBundle/Resources/app/bootstrap.php" colors="true">
+
+    <testsuites>
+        <testsuite name="SuluEventLogBundle Test Suite">
+            <directory suffix="Test.php">./Tests</directory>
+        </testsuite>
+    </testsuites>
+
+    <filter>
+        <whitelist>
+            <directory>./</directory>
+            <exclude>
+                <directory>./Tests</directory>
+                <directory>./vendor</directory>
+            </exclude>
+        </whitelist>
+    </filter>
+
+    <php>
+        <env name="APP_ENV" value="test"/>
```

Comment:
Normally is [forced](https://github.com/symfony/recipes/blob/4165f3f3d0e6ad6a5e70f15003ecd81d7e474fe8/phpunit/phpunit/4.7/phpunit.xml.dist#L12) for tests so eventual set APP_ENV can not accidently override it:

```suggestion
     <server name="APP_ENV" value="test" force="true" />
```

Reply:
changed for all bundles 🙂